src/opts/opts_lgrngn.hpp

```diff
@@ -254,6 +317,10 @@ void setopts_micro(
   
   rt_params.cloudph_opts_init.turb_adve_switch = vm["turb_adve"].as<bool>();
   rt_params.cloudph_opts.turb_adve = vm["turb_adve"].as<bool>();
+  
+  // set out_xxx_str from command line option input
+  rt_params.out_dry_str = vm["out_dry"].as<std::string>();
+  rt_params.out_wet_str = vm["out_wet"].as<std::string>();
```

Comment:
what are these needed for? 
One thing I can think of is to output them to const.h5

Reply:
Ah right. This is leftover from awhile ago. So I actually tried to output them in const.h5 and ran into a very annoying problem with hdf files which is that they have a limit on how long the output can be. With the experiments I was running, the number of bins I want to use results in a very long string and I got an error saying that this string was just too long. I came up with a silly work around where I just save this string as a separate .txt file from my slurm run script.

Anyways, I have removed these lines :)